tool/teleport/common/teleport.go

```diff
@@ -1127,3 +1138,29 @@ func onIntegrationConfAccessGraphAWSSync(params config.IntegrationConfAccessGrap
 
 	return nil
 }
+
+func onIntegrationConfSAMLIdPGCPWorkforce(params samlidpconfig.GCPWorkforceAPIParams) error {
+	ctx := context.Background()
+
+	// Ensure we print output to the user. LogLevel at this point was set to Error.
+	utils.InitLogger(utils.LoggingForDaemon, slog.LevelInfo)
+
+	// httpClient is used to fetch SAML IdP metadata.
+	httpClient, err := defaults.HTTPClient()
+	if err != nil {
+		return trace.Wrap(err)
+	}
```

Comment:
I still think this should move to CheckAndSetDefaults cause it sets a default client.

Reply:
https://github.com/gravitational/teleport/pull/39820/commits/07c52936198fcc729436fce7b3b6c5b8c81fe425